config/external_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"slices"
```

Comment:
This package was [introduced in go version `1.21.4`](https://pkg.go.dev/slices). We need to change `go.mod`:

https://github.com/cloudbase/garm/blob/main/go.mod#L3